### PR TITLE
feat(atoms): add blur content for modals

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/atoms/Modal/Modal.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ type ModalState = {
   blurred: boolean,
 };
 
+const MODAL_BODY_CLASS = 'modal-shown';
```
Comment (ej9x) [inline src/atoms/Modal/Modal.js]: Where are the styles of this class?

Comment (zouxuoz) [reply src/atoms/Modal/Modal.js]: @ej9x are you think we should hardcode styles with `.modal-shown > #page` selector?

Comment (ej9x) [reply src/atoms/Modal/Modal.js]: @zouxuoz I mean about adding blur css styles to the boost.

File: src/atoms/Modal/Modal.js [hunk 2]
```diff
@@ -74,21 +74,41 @@ class Modal extends PureComponent<ModalProps, ModalState> {
...
 
       if (typeof this.props.onClose === 'function') {
         this.props.onClose();
       }
+
+      this.updateBodyClass();
+    }
+  }
+
+  updateBodyClass() {
```
Comment (vorobeez) [inline src/atoms/Modal/Modal.js]: I think `upadteBodyClass` is meaningless function name. Can you add more specificity for this method?

Comment (zouxuoz) [reply src/atoms/Modal/Modal.js]: @vorobeez fixed

**Pull request merged**